app/src/androidTest/java/com/example/triptracker/map/AddSpotTest.kt

```diff
@@ -85,27 +85,26 @@ class AddSpotTest {
     Intents.release()
   }
 
-  /*
-    @Test
-    fun dropDownTestOk() {
-      ComposeScreen.onComposeScreen<AddSpotScreen>(composeTestRule) {
-        locationRow { assertIsDisplayed() }
-        locationText {
-          assertIsDisplayed()
-
-          performTextClearance()
-
-          performTextInput("ecole polytechnique federale")
-          composeTestRule.onNodeWithTag("LocationDropDown").performClick()
-        }
-        runBlocking { delay(2000) }
-        locationText {
-          assertIsDisplayed()
-          assertTextContains("École Polytechnique Fédérale de Lausanne", substring = true)
-        }
-      }
-    }
-  */
+  //    @Test
+  //    fun dropDownTestOk() {
+  //      ComposeScreen.onComposeScreen<AddSpotScreen>(composeTestRule) {
+  //        locationRow { assertIsDisplayed() }
+  //        locationText {
+  //          assertIsDisplayed()
+  //
+  //          performTextClearance()
+  //
+  //          performTextInput("ecole polytechnique federale")
+  //          composeTestRule.onNodeWithTag("LocationDropDown").performClick()
+  //        }
+  //        runBlocking { delay(2000) }
+  //        locationText {
+  //          assertIsDisplayed()
+  //          assertTextContains("École Polytechnique Fédérale de Lausanne", substring = true)
+  //        }
+  //      }
+  //    }
+
```

Comment:
Will this test be re-used later ? Or should we remove it ?

Reply:
This can still be used later ! For now it's crashing since nominatim is too slow (as in the bootcamp)